timesketch/api/v1/resources/upload.py

```diff
@@ -440,6 +450,14 @@ def post(self):
         if not form:
             form = request.form
 
+        # map existing headers with the mandatory ones
+        res = form.get('headersMapping')
```

Comment:
How about `res = form.get('headersMapping',None)` so that it is clear if the param does not exist it will be None?